src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -13979,5 +13948,196 @@ int32[] arg2
                 } // end of class <>f__AnonymousDelegate0
                 """);
         }
+
+        [Fact]
+        public void ParamsArray_InferredConversion()
```

Comment:
nit: I'm a bit confused by the names of some of the tests. For example, this test covers delegate conversions that include one anonymous delegate.

Reply:
Sorry about the confusion, some of it was probably caused by rewriting the test bodies without updating their names. Changed test names: bf5ad716b7054554352703f15d08a4c7d79b42fa

Although for example this test name was supposed to mean that we are testing conversion between two inferred (`var`) delegate types.